src/frontend/komponenter/Fagsak/TilkjentYtelse/Oppsummeringsrad.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import { IOppsummeringBeregning, ytelsetype } from '../../../typer/beregning';
+import Chevron from 'nav-datovelger/lib/elementer/ChevronSvg';
+import { datoformat, formaterBeløp, formaterIsoDato } from '../../../utils/formatter';
+import BeregningDetalj from './BeregningDetalj';
+import { Element, Normaltekst } from 'nav-frontend-typografi';
+import classNames from 'classnames';
+import { kategorier } from '../../../typer/behandling';
+
+interface IProps {
+    beregning: IOppsummeringBeregning;
+}
+
+const Oppsummeringsrad: React.FunctionComponent<IProps> = ({ beregning }) => {
+    const [åpentElement, setÅpentElement] = React.useState<boolean>(false);
+
+    const oppdaterÅpentElement = () => {
+        setÅpentElement(!åpentElement);
+    };
+
+    const kolonneClassnames = classNames('tilkjentytelse-kolonne', { åpen: åpentElement });
+    const radClassnames = classNames('tilkjentytelse-rad', { åpen: åpentElement });
+    const distinkteStønadstyper = [...new Set(beregning.stønadstype)];
+    return (
+        <div className={radClassnames} onClick={() => oppdaterÅpentElement()} role="button">
+            <div className={kolonneClassnames}>
+                <Chevron retning={åpentElement ? 'opp' : 'ned'} />
+            </div>
+            <div className={kolonneClassnames}>
+                <Normaltekst>
+                    {formaterIsoDato(beregning.periodeFom, datoformat.DATO_FORKORTTET)} -{' '}
+                    {formaterIsoDato(beregning.periodeTom, datoformat.DATO_FORKORTTET)}
```

Comment:
Kan dere bruke formaterPeriode (eller hva den heter) her?

Reply:
`periodeToString` du tenker på? Det kan vi prøve, men da bør den kunne ta inn datoformat ettersom vi ikke skal ha `YYYY`, men `YY`. 

Med mindre vilkårsvurderingen også kan bruke `YY` ?

Reply:
Ah, merkelig at man ønsker forskjellig datoformattering overalt :/ Kanskje vi kunne utvidet periodeToString til å ta inn format? Blir penere med felles-funksjonalitet for slike ting tenker jeg :)